test/test-markdown-external.js

```diff
@@ -0,0 +1,19 @@
+
+
+Reveal.addEventListener( 'ready', function() {
+
+	QUnit.module( 'Markdown' );
+
+	test( 'Vertical separator', function() {
+		strictEqual( document.querySelectorAll( '.reveal .slides>section>section' ).length, 2, 'found two slides' );
```

Comment:
Got it. Looking at it again I can see the vertical separator test was carried over from the `test-markdown.js` file, I'll revise it to match the simple.md that `test-markdown-external.js` tests against.